package.json

```diff
@@ -9,7 +9,7 @@
         "test": "karma start",
         "test:coverage": "rimraf coverage/ && karma start --single-run --browsers ChromeHeadless",
         "test:coveralls": "cat coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
-        "test:acceptance": "scripts/run-cypress-tests.sh $npm_package_versions_graphdb",
+        "test:acceptance": "export $(cat .env | xargs) && scripts/run-cypress-tests.sh",
```

Comment:
Just for the record, we can use https://www.npmjs.com/package/dotenv for dealing with .env files.

Reply:
Yes, we can. For now it's not really needed.